app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+
+
+def calculate_team_total_rating(players: list[Player]) -> None:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elfs: list[Elf]) -> None:
+    [elf.play_elf_song() for elf in elfs]
+
+
+def feast_of_the_dwarves(dwarfs: list[Dwarf]) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarfs]
```

Comment:
list comprehension is creating additional list here which is not good, 'cause of unneeded memory usage for no reason.
`for loop` is better here, since you only need to call functions

Reply:
+